drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -144,11 +144,20 @@ def items_without_answers():
                 item['inputOptions'] = 'inputOptions' in item
             return items
 
+        def title_with_points():
+            """
+            Build title using `display_name` and `weight` of this Drag and Drop exercise.
```

Comment:
What's the rationale for this change to the title? I'm not sure that course authors will like it in general - some drag and drop assignments will be ungraded, and it might confuse students to talk about how many points they can earn. In addition, authors can always just set a title like "Shapes Drag and Drop Exercise (1 point possible)", manually including the point value in the title string.